tests/data/dataset_readers/SQUAD_SAMPLE_FILE.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
But the spacing shouldn't matter for json processing, right?  And for really long strings that going into a `"""` block, you can just define them above, and reference them as variables.  Would that work?